modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/internal/jaxrs/application/FDSApplication.java

```diff
@@ -75,45 +73,6 @@
 )
 public class FDSApplication extends Application {
 
-	@DELETE
-	@Path("/fds/{fdsName}/custom-views/{fdsCustomViewId}")
-	public Response deleteFDSCustomView(
```

Comment:
Nice cleanup!

Removals make this a breaking change, but this feature is not used in DXP, and disabled by default, so we should be good. I highly doubt anybody would ever use this independently of FDS UI.

On the off chance a customer ever used custom views, they will be left with some harmless junk in portal preferences. I think this is acceptable, and we don't need to complicate into some DB migration or cleanup.